roles/tftp-server/templates/grub.cfg.j2

```diff
@@ -0,0 +1,76 @@
+# /srv/tftp/grub/grub.cfg
+
+timeout=5
+
+# get extra appends from dhcp
+# allows per host config options
+net_get_dhcp_option appends ${net_default_interface} 209 string
+# spell hostname funny else it collides with, I guess hostname
+net_get_dhcp_option hstnam ${net_default_interface} 12 string
+
+tftp=${pxe_default_server}
+
+shaz=${tftp}
```

Comment:
poorly named variable

Reply:
I know what it means :D 
hostname of first tftp server I ever setup.

I ran out of good names.

this is another hook to send all http stuff to another box for hacking.
hmm, http=${pxe_default_server}
seems odd.  
got a good name?

Reply:
why do we even need two variables, isn't $tftp good enough?

Reply:
hey look at that.  
fixed.